apache-kafka/src/main/java/com/baeldung/flink/model/InputMessage.java

```diff
@@ -55,13 +56,12 @@ public InputMessage(String sender, String recipient, LocalDateTime sentAt, Strin
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
         InputMessage message1 = (InputMessage) o;
-        return Objects.equal(sender, message1.sender) &&
-                Objects.equal(recipient, message1.recipient) &&
-                Objects.equal(sentAt, message1.sentAt) &&
-                Objects.equal(message, message1.message);
+        return Objects.equal(sender, message1.sender) && Objects.equal(recipient, message1.recipient) && Objects.equal(sentAt, message1.sentAt) && Objects.equal(message, message1.message);
```

Comment:
Let's keep the original formatting - each Objects.equal in its own line.

Reply:
Done, please check!